Pages/Hotels/HotelHomePage.js

```diff
@@ -135,6 +147,96 @@ async isRemoveChildBttnEnabled() {
     return removeChildBttn.isEnabled();
 }
 
+async selectEspañol() {
+    const langDropDown = await $(this.languageBoxLocator);
+    await langDropDown.selectByIndex(1);
+    await browser.pause(2000);
+}
+
+async selectEnglish() {
+    const langDropDown = await $(this.languageBoxLocator2);
+    await langDropDown.selectByIndex(0);
+    await browser.pause(2000);
```

Comment:
pause() isn't efficient way for waiting, it'll just stop execution for few sec then it'll start execution again doesn't matter whether the required element is there or not. So we should use waitForDisplayed() or waitForEnabled(). If the element isn't displayed test gonna fail. And that is professional.